include/cutlass/arch/mma.h

```diff
@@ -68,14 +68,24 @@ struct OpMultiplyAddFastF16 {};
 
 /////////////////////////////////////////////////////////////////////////////////////////////////
 
+/// Tag indicating the input data types are mixed and the narrower type is 
+/// converted to the wider type
+struct OpMultiplyAddMixedInput {};
```

Comment:
Can we call this `OpMultiplyAddMixedInputUpcast`? It should leave room if we implement a version that converts the wider type to the narrower type

Reply:
Thanks, this makes sense. I have updated the name.